scripts/update_bootlint.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+/* eslint-env shelljs */
+
+'use strict';
+
+require('shelljs/global');
+var https       = require('https');
+var fs          = require('fs');
+var path        = require('path');
+
+var version     = process.argv[2];
+
+// strip leading 'v' if present
+version = version.replace(/^v/, '');
+
+
+var basedir     = path.join(__dirname, '..');
+var bootlintDir = path.join(basedir, 'public', 'bootlint', version);
+var UGLIFYJS    = path.join(basedir, 'node_modules/.bin/uglifyjs');
+
+if (!version) {
+    echo('Valid Bootlint version required.');
+    process.exit(1);
+}
+
+if (test('-d', bootlintDir)) {
+    echo('Bootlint version already found.');
+    process.exit(1);
+}
+
+https.get('https://raw.githubusercontent.com/twbs/bootlint/v' + version + '/dist/browser/bootlint.js', function (res) {
```

Comment:
Since this is more of a script, might this be a good use-case for `await https.get(uri)`?